index.html

```diff
@@ -41,6 +47,103 @@ <h3 class="supporting-text">Software Engineering</h3>
         </div>
         <div class="vector4"></div>
       </section>
+      <section class="section-2">
+        <div class="language">
+          <h2>Language</h2>
+          <p>Javascript</p>
+          <p>Ruby</p>
+          <p>Node</p>
+          <p>CSS</p>
+          <p>Html</p>
+        </div>
+
+        <div class="framewoks">
+          <h2>Frameworks</h2>
+          <p>Bootstrap</p>
+          <p>Ruby ob Rails</p>
+          <p>RSPec</p>
+          <p>CapyBara</p>
+          <p>Selenium</p>
+        </div>
+
+        <div class="skills">
+          <h2>Skills</h2>
+          <p>Codekit</p>
+          <p>GitHub</p>
+          <p>Codepen</p>
+          <p>Gitlab</p>
+          <p>Terminal</p>
+        </div>
+        <div class="vector5"></div>
+      </section>
+
+      <div class="icons">
+        <i class="fa-brands fa-github"></i>
+        <i class="fab fa-linkedin-in"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fab fa-medium"></i>
+      </div>
+
+      <section class="my-work">
+        <div class="work-tag"><i class="fa-thin fa-arrow-down"></i>Work</div>
+
+        <h1>MY RECENT WORK</h1>
+
+        <div class="vector6"></div>
+
+        <div class="my-work-1">
+          <div class="img-placeholder"></div>
+          <h2 class="work-title">Multi-Post Stories</h2>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
```

Comment:
- [ ] Please refer to this [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design to match the font-size and font-weight of the original design, see the images below, and please adjust them all in accordance with your original Figma design.
<img width="321" alt="grb yuuuuu" src="https://user-images.githubusercontent.com/79058364/153729925-99a1310e-81b8-4ddf-b514-33fd0076d16d.png">
instead, it should display like this 👍🏻  below ⬇️ 
<img width="256" alt="grnffff" src="https://user-images.githubusercontent.com/79058364/153729948-f7d30bd5-1a54-46f0-b1e5-9a8ae4a26789.png">